ignore.go

```diff
@@ -0,0 +1,44 @@
+package git
+
+/*
+#include <git2.h>
+*/
+import "C"
+import (
+	"runtime"
+)
+
+func (v *Repository) AddInternalIgnoreRules(rules string) error {
```

Comment:
[This document](https://libgit2.github.com/libgit2/#HEAD/group/ignore/git_ignore_add_rule) says that the parameter `rules` can have multiple rules in it.
Anyway, I will change it to `AddIgnoreRule(rules string)` to match libgit2.